PixelDefinitions/suffixes_dictionary.json

```diff
@@ -0,0 +1,25 @@
+// This file defines commonly used suffixes - this does NOT mean they are added by default
+// In your pixel definitions, you can reference these suffixes by their key
+// See TEMPLATE.json for an example
+{
+    "first_daily_count": {
+        "type": "string",
+        "description": "Pixel schedule type",
+        "enum": ["first", "daily", "count"]
+    },
+    "form_factor": {
+        "type": "string",
+        "description": "Device form-factor",
+        "enum": ["phone", "tablet"]
+    },
+    "platform": {
+        "type": "string",
+        "description": "Device platform",
+        "enum": ["android"]
+    },
+    "channel": {
```

Comment:
Similar to other comments: is `channel` utilized on Android? If not, we should remove from here, params_dictionary and TEMPALTE.